### PR TITLE
[multiple_web] Update web plugin testing instructions.

### PR DESCRIPTION
**Pull request by NishantChandla**
Updated web testing instructions in example/README.md for web plugins. Latest web-test instructions point to https://github.com/flutter/flutter/wiki/Plugin-Tests#web-tests.

* Fixes https://github.com/flutter/flutter/issues/76817

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Note that unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I [updated pubspec.yaml](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

### CODE REVIEW

File: packages/url_launcher/url_launcher_web/example/README.md [hunk 1]
```diff
@@ -1,24 +1,12 @@
...
-3. Start the driver using `chromedriver --port=4444`
-
-4. Run tests: `flutter drive -d web-server --browser-name=chrome --driver=test_driver/integration_test_driver.dart --target=integration_test/TEST_NAME.dart`, or (in Linux):
-
-    * Single: `./run_test.sh integration_test/TEST_NAME.dart`
-    * All: `./run_test.sh`
+Check [flutter.dev > Integration testing](https://flutter.dev/docs/testing/integration-tests)
+for more info.
 
 ## Mocks
```
Comment (ditman) [inline packages/url_launcher/url_launcher_web/example/README.md:11]: Mocks are also included in the Wiki above. Let's replace the whole  contents of this section by:

```
See [Plugin Tests > Web Tests > Mocks](https://github.com/flutter/flutter/wiki/Plugin-Tests#mocks)
in the Flutter wiki for more information about the `.mocks.dart` files in this package.
```

(This also happens in the `google_maps_flutter_web` package)

**Review by ditman — CHANGES_REQUESTED** [review]
Please, remove the "hardcoded" documentation about  Mocks in url_launcher_web and google_maps_flutter_web, and point people to the flutter wiki as well. I've left a small example markdown that should work!

And thanks for your contribution!

**Review by ditman — APPROVED** [review]
Looking great, I'll have this merged ASAP!

**Comment by NishantChandla** [comment]
> Looking great, I'll have this merged ASAP!

Thank You! @ditman

**Comment by vishnuaq** [comment]
Shh

**Review by vishnuaq — COMMENTED** [review]
**bsnxn**

**Pull request merged**